Cargo.toml

```diff
@@ -18,6 +18,7 @@ spirv = { package = "spirv_headers", version = "1.4.2", optional = true }
 glsl = { version = "4.1", optional = true }
 pomelo = { version = "0.1.4", optional = true }
 thiserror = "1.0"
+daggy = "*"
```

Comment:
let's not do wildcard dependencies :)

Reply:
Also, it should be totally optional, behind the `spirv` feature. Currently, `spirv` the feature comes from the optional package. I think we'd want to avoid renaming `spirv_headers` to `spirv` here in the manifest, and just have it specified as:
```toml
spirv = ["daggy", "spirv_headers"]
```